packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -40,83 +40,36 @@ function BlockStylesPanel( { clientId } ) {
 	);
 }
 
-function BlockInspectorLockedBlocks( { topLevelLockedBlock } ) {
-	const contentClientIds = useSelect(
-		( select ) => {
-			const {
-				getClientIdsOfDescendants,
-				getBlockName,
-				getBlockEditingMode,
-			} = select( blockEditorStore );
-			return getClientIdsOfDescendants( topLevelLockedBlock ).filter(
-				( clientId ) =>
-					getBlockName( clientId ) !== 'core/list-item' &&
-					getBlockEditingMode( clientId ) === 'contentOnly'
-			);
```

Comment:
cc @fabiankaegy, it looks to me like the workaround was in place already before this PR was merged, I think all it does is hide list item blocks in the 'Content' panel.

Reply:
🥸 Okay my bad. In that case I have yet to find where we are actually solving it 😅Sorry for my confusion!